locales/da.json

```diff
@@ -6,46 +6,47 @@
     "addkeys_id_help": "Spiller ID",
     "addkeys_plate": "nummerplade",
     "addkeys_plate_help": "Nummerplade",
-    "givekeys": "givede nøgler",
-    "givekeys_help": "Overlever nøglerne til nogen. Hvis der ikke er nogen ID, gives de til den nærmeste person eller alle i køretøjet.",
+    "givekeys": "givnøgler",
```

Comment:
How about using “o” instead of “ø”?

Reply:
its a danish letter, pronounced "oeeh"

Reply:
I suggested this to simplify the command writing

Reply:
well, we could make a lookalike with a capital Q, but its a mess. So ty for letting Ø, Æ and Å into the mix <3